scripts/parsecandump.py

```diff
@@ -0,0 +1,125 @@
+#!/usr/bin/python3
+# Check for out of order timestamps in the output of candump
+#
+# Copyright (C) 2023  Kevin O'Connor <kevin@koconnor.net>
+#
+# This file may be distributed under the terms of the GNU GPLv3 license.
+import sys, os, optparse
```

Comment:
from the python docs:

```
Deprecated since version 3.2: The [optparse](https://docs.python.org/3/library/optparse.html#module-optparse) module is deprecated and will not be developed further; development will continue with the [argparse](https://docs.python.org/3/library/argparse.html#module-argparse) module.
```

Please use argparse instead.